cynic-codegen/src/enum_derive/mod.rs

```diff
@@ -38,7 +39,11 @@ pub fn enum_derive_impl(
 
     let enum_def = schema.definitions.iter().find_map(|def| {
         if let Definition::TypeDefinition(TypeDefinition::Enum(e)) = def {
-            if e.name == *input.graphql_type {
+            if e.name
+                == *input
+                    .graphql_type
+                    .extract_spanned_value(input.ident.to_string())
```

Comment:
If you take my `fn graphql_type_name` suggestion this can just become

`e.name == input.graphql_type_name()`